scripts/perf/bin/s2n-quic

```diff
@@ -12,7 +12,9 @@ case "$PS" in
           ../../target/release/s2n-quic-qns \
             perf \
             server \
-            --port $SERVER_PORT
+            --port $SERVER_PORT \
+            --max-mtu 16000 \
```

Comment:
Why is this 16000?

Reply:
It's over localhost, which technically supports 64k MTUs. But after playing around with some values it produced the best perf overall.